src/plugins/kibana/public/dashboard/index.js

```diff
@@ -100,7 +99,7 @@ app.directive('dashboardApp', function (Notifier, courier, AppState, timefilter,
       });
       $scope.$watch('state.options.darkTheme', setDarkTheme);
 
-      $scope.configTemplate = new ConfigTemplate({
+      $scope.configTemplates = {
         save: require('plugins/kibana/dashboard/partials/save_dashboard.html'),
```

Comment:
Can we get rid of the `configTemplates` and just move each one to a template property on `topNavMenu`?